src/main/java/domain/Command.java

```diff
@@ -1,32 +1,41 @@
 package domain;
 
 import java.util.Arrays;
+import java.util.Objects;
 import java.util.StringJoiner;
 
 public enum Command {
     ALL("all"),
     EXIT("exit");
 
-    private final String command;
+    private final String value;
 
-    Command(String command) {
-        this.command = command;
+    Command(String value) {
+        this.value = value;
     }
 
-    public static boolean contains(String string) {
+    public static boolean contains(String input) {
         return Arrays.stream(Command.values())
-                .anyMatch(e -> e.command.equals(string));
+                .anyMatch(e -> e.value.equals(input));
+    }
+
+    public static boolean isAll(String input) {
+        return Objects.equals(input, ALL.getValue());
```

Comment:
개인적인 경험으론 Objects.equals를 사용할 일이 별로 없는 것 같아요. 
오히려 equals를 재정의해 객체 그자체의 equals를 사용하는게 더 좋아보입니다.
또한 현재 코드는 아래와 같이 작성할 수 있습니다 :)
```suggestion
        return ALL.value.equals(input);
```